Source/WebCore/Modules/webauthn/fido/Pin.h

```diff
@@ -138,6 +137,7 @@ struct SetPinRequest {
 public:
     WEBCORE_EXPORT const WebCore::CryptoKeyAES& sharedKey() const;
     WEBCORE_EXPORT static std::optional<SetPinRequest> tryCreate(const String& newPin, const WebCore::CryptoKeyEC&);
+    WEBCORE_EXPORT Vector<uint8_t> pinAuth() const;
```

Comment:
If `pinAuth()` is backed by a Vector member, why isn't it returned by reference so we can avoid building a Vector for most uses?